src/components/cms/Dashboard/MySkills.js

```diff
@@ -45,7 +45,7 @@ const Container = styled.div`
     margin-right: 0;
   }
 `;
-
+const issmallscreen = window.screen.availWidth < 500;
```

Comment:
Use camel case for variable naming. `isSmallScreen`

Reply:
I used _(underscores) before linters give errors on that. I should have used CamelCase.. Thanks for pointing that out. Will Do that change